linux_os/guide/system/accounts/accounts-restrictions/root_logins/ensure_pam_wheel_group_empty/rule.yml

```diff
@@ -2,18 +2,17 @@ documentation_complete: true
 
 prodtype: rhel8,rhel9,sle12,sle15,ubuntu2004,ubuntu2204
 
-title: 'Ensure the Group Used by pam_wheel Module Exists on System and is Empty'
+title: 'Ensure the Group Used by pam_wheel.so Module Exists on System and is Empty'
 
 description: |-
-    Ensure that the group {{{ xccdf_value("var_pam_wheel_group_for_su") }}}
-    referenced by the pam_wheel  <tt>group</tt> parameter exists and has no
-    members. This ensures that no user can run commands with altered
-    privileges through the <tt>su</tt> command.
+    Ensure that the <tt>{{{ xccdf_value("var_pam_wheel_group_for_su") }}}</tt> group referenced by
+    the <tt>pam_wheel.so</tt> <tt>group</tt> option exists and has no members. This ensures
+    that no user can run commands with altered privileges through the <tt>su</tt> command.
```

Comment:
I still find this slightly confusing. My confusion comes from the fact that here we talk about `pam_wheel.so` module option, but this rule's OVAL and OCIL don't check any PAM configuration, it only checks the existence of a group in `/etc/group`. From this context the group name might seem arbitrary. Apparently, the PAM configuration is handled by rule `use_pam_wheel_group_for_su` and both rules are selected together to provide an experience. But If I imagine that I'm a user that suddenly sees this rule failing I think i might not be aware of this connection with the other rule. I'm not sure how to reword this description to reflect that.

Reply:
@jan-cerny could you take a look in the last two commits, please?